modules/Xsens_module/src/XsensRetargeting.cpp

```diff
@@ -34,6 +35,15 @@ bool XsensRetargeting::configure(yarp::os::ResourceFinder& rf)
         return false;
     }
 
+    // check if doing yoga retargeting, otherwise it is Xsens-walking retargeing
+    m_yogaRetargeting = rf.check("yogaRetargeting", yarp::os::Value(false)).asBool();
+
+    // check if use the smoothing, otherwise we do not smooth the joint values
+    m_useSmoothing = rf.check("useSmoothing", yarp::os::Value(true)).asBool();
+
+    yInfo() << "[XsensRetargeting::configure] m_yogaRetargeting: " << m_yogaRetargeting;
+    yInfo() << "[XsensRetargeting::configure] m_useSmoothing: " << m_useSmoothing;
```

Comment:
What's the difference between using `yogaRetargeting` and not using it?

Reply:
The joint list size and order might be different. For example in yoga retargeting we do not consider the neck joint values.

Reply:
the difference is in joint lists, as you suggested later we can use two different configuration files instead which is more clean.